Header header
uint8 slave_id
bool state_machine
bool initialize_clock
bool initialize_angles
bool inverse_kinematics
bool blue_led
bool red_led
bool button_1
bool button_2
int8 sync
int32 desired_x_value
uint16 filter_bandwidth
int32 desired_y_value
int16 kp_100_knee
int16 kd_1000_knee
int16 ki_100_knee
int16 kp_100_hip
int16 kd_1000_hip
int16 ki_100_hip
int16 x_cntr_traj1000
int16 y_cntr_traj1000
int16 a_ellipse100
int16 b_ellipse100
int16 traj_freq100
int16 phase_deg
int16 flatness_param100
